examples/misc/raptor.py

```diff
@@ -1,4 +1,29 @@
 #!/usr/bin/env python3
+"""Demonstrate the "raptor" features for remote Task management.
+
+This script and its supporting files may use relative file paths. Run from the
+directory in which you found it.
+
+Refer to the ``raptor.cfg`` file in the same directory for configurable run time details.
+
+By default, this example uses the ``local.localhost`` resource with the ``local``
+access scheme. Before running, confirm that you are able to ssh to localhost
+without being prompted for a password.
+
+In this example, we
+* Launch one or more raptor "master" task(s), which self-submits additional tasks.
+  (TODO: How do we check/confirm their results?)
```

Comment:
We don't really need to as this is not a stringent test, but in principle the test would go into the master's `result_cb`.

Reply:
If someone running the example is not supposed to have any sort of run time confirmation that the script did what it was supposed to, then there should at least be some text describing what to look for.

It would also be helpful if the examples could be used as integration tests, or to validate a RP installation, like for https://github.com/radical-cybertools/radical.pilot/tree/devel/docker or to decrease the overhead of testing dependent projects like scalems. For the purposes of the current PR, I recognize that this is not currently a condoned use case for the example scripts. But these clarifications should help downstream projects to better design such tests.